Api/KanbanizeApi.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Chaplean\Bundle\KanbanizeApiClientBundle\Api;
+
+use Chaplean\Bundle\ApiClientBundle\Api\AbstractApi;
+use Chaplean\Bundle\ApiClientBundle\Api\Parameter;
+use Chaplean\Bundle\ApiClientBundle\Api\Route;
+use GuzzleHttp\ClientInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * Class KanbanizeApi.
+ *
+ * @method Route getChildCards()        Get a list of child cards                  ( #/Child_Cards/getChildCards )
+ * @method Route putChildCard()         Make a card a child of a given card        ( #/Child_Cards/addChildCard )
+ * @method Route getParentCards()       Get a list of parent cards                 ( #/Parent_Cards/getParentCards )
+ * @method Route putParentCard()        Make a card a parent of a given card       ( #/Parent_Cards/addParentCard )
+ * @method Route getPredecessorCards()  Get a list of predecessor cards            ( #/Predecessor_Cards/getPredecessorCards )
+ * @method Route putPredecessorCard()   Make a card a predecessor of a given card  ( #/Predecessor_Cards/addPredecessorCard )
+ * @method Route getSuccessorCards()    Get a list of successor cards              ( #/Successor_Cards/getSuccessorCards )
+ * @method Route putSuccessorCard()     Make a card a successor of a given card    ( #/Successor_Cards/addSuccessorCard )
+ *
+ * @package   Chaplean\Bundle\KanbanizeApiClientBundle\Api
+ * @author    Valentin - Chaplean <valentin@chaplean.coop>
+ * @copyright 2014 - 2018 Chaplean (http://www.chaplean.coop)
+ */
+class KanbanizeApi extends AbstractApi
```

Comment:
Wouldn't specify v2 be clearer and prevent breaking changes issues the day a potential v3 comes around?